src/Domain/CoordinateMaps/CubicScale.hpp

```diff
@@ -28,66 +28,78 @@ namespace domain {
 namespace CoordMapsTimeDependent {
 /*!
  * \ingroup CoordMapsTimeDependentGroup
- * \brief CubicScale map defined by \f$x = a(t)*\xi+(b(t)-a(t))\xi^3/X^2\f$,
- *  where \f$X\f$ is the outer boundary.
+ * \brief Maps the radius \f$r\f$ as \f$r(t) = a(t)\rho + \left(b(t) - a(t)
```

Comment:
The help text would be clearer to me if you included the formula for the mapped coordinates \bar{x}^i in terms of the source coordinates x^i

Reply:
That's on line 42...

Reply:
Wasn't there in my version...must have pulled before your fixup?